src/test/resources/pages/InstructorCourseDetailsEmptyCourse.html

```diff
@@ -1,5 +1,8 @@
 <html>
 <head>
+<style id="mceDefaultStyles" type="text/css">
+  .mce-content-body div.mce-resizehandle {position: absolute;border: 1px solid black;background: #FFF;width: 5px;height: 5px;z-index: 10000}.mce-content-body .mce-resizehandle:hover {background: #000}.mce-content-body img[data-mce-selected], hr[data-mce-selected] {outline: 1px solid black;resize: none}.mce-content-body .mce-clonedresizable {position: absolute;opacity: .5;filter: alpha(opacity=50);z-index: 10000}.mce-content-body .mce-resize-helper {background: #555;background: rgba(0,0,0,0.75);border-radius: 3px;border: 1px;color: white;display: none;font-family: sans-serif;font-size: 12px;white-space: nowrap;line-height: 14px;margin: 5px 10px;padding: 5px;position: absolute;z-index: 10001} img:-moz-broken {-moz-force-broken-image-icon:1;min-width:24px;min-height:24px}
+</style>
```

Comment:
These lines will be ignored anyway, so can you remove them? Similarly for the other files.